### ISSUE TITLE
Missing style error.

### ISSUE THREAD

**Issue by jch02140** [mars-x24/Marketplace#1]
I am getting a missing style reference error when I log into server of cryofall...

![error](https://user-images.githubusercontent.com/935533/120107127-5af00f00-c192-11eb-99af-7c6d513a5d15.jpg)

**Comment by aienabled** [comment]
Yes, please fix this as it has various side effects on the UI and may lead to completely broken UI (requiring the game to restart).
This line should be removed https://github.com/mars-x24/Marketplace/blob/master/UI/Marketplace/Styles/DefaultStyles.xaml#L12
I'm not sure but probably these styles are already linked in DefaultStyles.xaml

Regards!
